posydon/popsyn/population_params_default.ini

```diff
@@ -212,7 +212,13 @@
     # 'N20' for 'Sukhbold+16-engine',
     # 'Patton&Sukhbold20-engine' or None for the others
   PISN = "Marchant+19"
-    # None, "Marchant+19"
+    # None, "Marchant+19", "Hendriks+23"
+  PISN_CO_shift = 0.0
+    # float (-inf,inf)
+    # Only when using Hendriks+23
+  PPI_extra_mass_loss = 0.0
+    # float (0,inf)
```

Comment:
At the moment the code doesn't ensure that the extra mass loss is not negative. May add a check for this in the code.